sdk/servicebus/service-bus/test/utils/envVarUtils.ts

```diff
@@ -46,10 +41,6 @@ export function getEnvVars(): { [key in EnvVarNames]: any } {
 
   let serviceBusConnectionStringEnvVarName: string = EnvVarNames.SERVICEBUS_CONNECTION_STRING.valueOf();
 
-  if (!isNode) {
-    serviceBusConnectionStringEnvVarName += "_BROWSER";
```

Comment:
Now that we are not modifiying the env var name for browser, there is no need to have the `serviceBusConnectionStringEnvVarName` variable.